### PR TITLE
test(terraform/lxd): fix lxd terraform provider

### PR DESCRIPTION
**Pull request by tiagolobocastro**
ci(events): disable events test until fixed
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    fix(composer): capture logs from failed tests
    
    At the same time, ensure they don't get printed twice.
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    ci(gha/bors): run commit and submodule check on the staging branch
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    fix(pool/destroy): destroy when not found
    
    When the pool is not found, also allow destroying it.
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    ci: pass DCO on staging
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    test(terraform/lxd): fix lxd terraform provider
    
    On LXD, we can't set affinity for CPU's as the io-engine tries to bind to specific cpu's,
     so this won't work..
    We need to come up with a new, more flexible way to deploy the io-engine and bind to cpus.
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

File: .github/workflows/staging-dco.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  push:
+    branches:
+      - staging
+
+jobs:
+  DCO:
+    runs-on: ubuntu-latest
+    steps:
+      - name: DCO
+        run: echo "DCO"
```
Comment (Abhinandan-Purkait) [inline .github/workflows/staging-dco.yml:12]: Question: Is this a hack to get round DCO?

**Review by Abhinandan-Purkait — APPROVED** [review]

File: .github/workflows/staging-dco.yml (hunk 1 above)
Comment (tiagolobocastro) [reply .github/workflows/staging-dco.yml:12]: Yes indeed, as DCO fails when bors has to rebase

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors cancel

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**